common/components/table/template.njk

```diff
@@ -0,0 +1,2 @@
+{% from "govuk/components/table/macro.njk"             import govukTable %}
+{{ govukTable(params) }}
```

Comment:
It feels like we could remove this component as its doing the same as the GOV.UK one? Then just call `govukTable()` in your view rather than `appTable()`?

Reply:
This is the first iteration, I fully expect us to extend (or even fork it) this component soon, hence my creating a place for it.

Reply:
Are we planning to use it anywhere else? Will it incorporate additional pagination elements and/or need to include additional css/js for the added functionality?

Right now though, it seems that the presenter is doing the heavy lifting (and will continue to do so?) and it is currently just a `govukTable`

Reply:
It will very likely incorporate other elements; pagination is certainly one.

Reply:
I'd say we park it for now until that case arises